packages/cli-core/src/features/DeployCommand/DeployCommand.ts

```diff
@@ -43,15 +49,18 @@ export class DeployCommand implements Command.Interface<IDeployCommandParams> {
             description: "Deploys specified app or all apps in the project",
             examples: [
                 "$0 deploy api --env dev",
+                "$0 deploy core api --env dev",
                 "$0 deploy admin --env prod",
                 "$0 deploy --env prod",
                 "$0 deploy"
             ],
             params: [
                 {
-                    name: "app",
-                    description: "Name of the app (core, admin, or api)",
-                    type: "string"
+                    name: "apps",
```

Comment:
I'd rather have a single `app` parameter, that is a single app or an array of apps. Or maybe even... Let's have it always be an array? Like `apps: AppName[]`.